2.exact_divisor.py

```diff
@@ -0,0 +1,7 @@
+def exact_divisor(number):
+
+    return [i for i in range(1, number + 1) if number % i == 0]
+    # We use list comprehension recording numbers that can divide the given number.
+
+
+print(*exact_divisor(12))
```

Comment:
The reason I put * was for formatting purposes. The output format in the assignment paper was numbers without parentheses or commas for some of the questions. When you put * front of the function you get a list of numbers without them. That is why I used *, not because of args or kwargs.